core/openapi/schemas/recordGeoJSON.yaml

```diff
@@ -23,7 +23,7 @@ properties:
     $ref: time.yaml
   geometry:
     oneOf:
-      - type: null
+      - type: "null"
```

Comment:
This is an OpenAPI 3.0 schema according to the path, so you cannot use `null` as a type. The property needs to be annotated as `nullable: true` instead.

Reply:
thanks @cportele.  Fixed.